src/components/HomeViewUsers/users.css

```diff
@@ -0,0 +1,45 @@
+img {
```

Comment:
I don't agree with the usage of global tag-reference.

Change this to BEM class names.